r/R/trackplots.R

```diff
@@ -450,6 +478,126 @@ trackplot_gene <- function(transcripts, region, exon_size = 2.5, gene_size = 0.5
   wrap_trackplot(plot, height=ggplot2::unit(1, "null"))
 }
 
+#' Plot peak tracks
+#' @param loci `r document_granges("Genomic loci")`
+#' @inheritParams trackplot_coverage
+#' @param annotation_size size for peak annotations
+#' @param label_by Name of a metadata column in `loci` to use for labeling, or a numeric vector with same length as loci  Column must be type character, factor or numeric
+#' @param label_size size for transcript labels in units of mm
+#' @param color_by Name of a metadata column in `loci` to use for coloring, or a numeric vector with same length as loci.  Column must be type character, factor, or numeric
+#' @param colors Vector of hex color codes to use for the color gradient
+#' @param direction_by Name of a metadata column in `loci` to use for determining direction of arrows
+#' @param arrow_angle Angle of the arrow head in degrees, if direction_by is provided
+#' @param track_label Label for the track
+#' @param return_data If TRUE, return the data frame used to generate the plot
+#' @return Plot of genomic loci if return_data is FALSE, otherwise returns the data frame used to generate the plot
+#' @seealso `trackplot_combine()`, `trackplot_coverage()`, `trackplot_loop()`, `trackplot_scalebar()`, `trackplot_gene()`
+#' @export
+trackplot_genome_annotations <- function(loci, region, annotation_size = 4,  label_by = NULL, label_size = 11*.8/ggplot2::.pt, 
```

Comment:
We should probably change this to singular `trackplot_genome_annotation` to match the convention of the other function names